node/parachain/Cargo.toml

```diff
@@ -62,22 +63,22 @@ pallet-transaction-payment-rpc = { git = "https://github.com/paritytech/substrat
 sp-block-builder = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sc-consensus-manual-seal = { git = "https://github.com/paritytech/substrate", branch = "master" }
 
-evm = { package = "pallet-evm", git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
-ethereum = { package = "pallet-ethereum", git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
+evm = { package = "pallet-evm", git = "https://github.com/purestake/frontier", branch = "v0.6-moonbeam" }
+ethereum = { package = "pallet-ethereum", git = "https://github.com/purestake/frontier", branch = "v0.6-moonbeam" }
 
-fc-consensus = { git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
-fp-consensus = { git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
-fc-rpc-core = { git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
-fc-rpc = { git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
+fc-consensus = { git = "https://github.com/purestake/frontier", branch = "v0.6-moonbeam" }
+fp-consensus = { git = "https://github.com/purestake/frontier", branch = "v0.6-moonbeam" }
+fc-rpc-core = { git = "https://github.com/purestake/frontier", branch = "v0.6-moonbeam" }
+fc-rpc = { git = "https://github.com/purestake/frontier", branch = "v0.6-moonbeam" }
 
 author-inherent = { path = "../../pallets/author-inherent"}
 
 # Cumulus dependencies
-cumulus-consensus = { git = "https://github.com/paritytech/cumulus", branch = "master" }
-cumulus-collator = { git = "https://github.com/paritytech/cumulus", branch = "master" }
-cumulus-network = { git = "https://github.com/paritytech/cumulus", branch = "master" }
-cumulus-primitives = { git = "https://github.com/paritytech/cumulus", branch = "master" }
-cumulus-service = { git = "https://github.com/paritytech/cumulus", branch = "master" }
+cumulus-consensus = { git = "https://github.com/paritytech/cumulus", rev = "6438d32" }
```

Comment:
Why we don't use the master branch here and lock the rev in the Lock file ?

Reply:
Fixed in 3418eca